fatukunda/go-party
# #168072057 User should view a single party

[Finishes: #168072057]

#### What does this PR do?
 - Enables a user to view a single party.
#### Description of Task to be completed?
 - Create a controller, service and route for viewing a single party
 - Write tests
#### How should this be manually tested?
 - Clone the repo by running `git clone https://github.com/fatukunda/go-party.git`.
 - Install dependencies by running `yarn install`.
 - Run the server by running `yarn dev`.
 - Navigate to `POST localhost:8000/api/v1/parties/:party_id` to view a single party.
 - If all goes well, You should view that party.
#### What are the relevant pivotal tracker stories?
[#168072057](https://www.pivotaltracker.com/story/show/168072057)
#### Any background context you want to add?
 - None
#### Screenshots
<img width="1132" alt="Screen Shot 2019-09-23 at 12 28 38 PM" src="https://user-images.githubusercontent.com/4363632/65415103-bbe01500-ddfd-11e9-8c0a-003cbdbdd933.png">
